Float.Core/Net/RequestClient.cs

```diff
@@ -295,5 +330,18 @@ async Task<Response> Send(HttpRequestMessage request)
                 throw;
             }
         }
+
+        /// <summary>
+        /// Unsuccess redirect exception helper to handle a content redirect.
+        /// </summary>
+        internal class UnsuccessRedirectException : Exception
```

Comment:
@arctouch-alejandrovarela Would you mind renaming this exception to something like `RequestRedirectedException`?

Reply:
done